MiniApp/Classes/MiniAppClient.swift

```diff
@@ -84,14 +92,14 @@ class MiniAppClient: NSObject, URLSessionDownloadDelegate {
         case 401, 403:
             guard let errorModel = ResponseDecoder.decode(decodeType: UnauthorizedData.self,
                                                           data: responseData) else {
-                                                            return NSError.unknownServerError(httpResponse: httpResponse)
+                return NSError.unknownServerError(httpResponse: httpResponse)
             }
             message = "\(errorModel.error): \(errorModel.errorDescription)"
         default:
             guard let errorModel = ResponseDecoder.decode(decodeType: ErrorData.self,
                                                           data: responseData) else {
-                    return NSError.unknownServerError(httpResponse: httpResponse)
-                }
```

Comment:
Unwanted spaces on new lines were making me crazy because Xcode automatically adds spaces on new lines and SwiftLint don't want them, so I installed a plungin in Xcode that reformat the code so it is 100% swift lint compliant. These were bad formatted lines so I thought it would make no harm if I committed them